### PR TITLE
[4.0] Add tooltip for workflow list view text-search box #pbf19 #codepatch

### PR DESCRIPTION
**Pull request by manojLondhe**
### Summary of Changes
Adds tooltip to text-search box

### Testing Instructions
1. Apply patch
2. Verify if tooltip shown for search box

### Expected result
Tooltip shown for search box

### Actual result
Tooltip shown for search box

### Documentation Changes Required
No

### CODE REVIEW

**Comment by vaibhavsTekdi** [comment]
#pbf19 #test1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26910">issues.joomla.org/tracker/joomla-cms/26910</a>.</sub>

**Comment by vaibhavsTekdi** [comment]
I have tested this item :white_check_mark: successfully on 39b5ba8a8b5c57c7e542570068f8df53d351a625<br /><br />#pbf19 #test1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26910">issues.joomla.org/tracker/joomla-cms/26910</a>.</sub>

**Comment by kishorchate** [comment]
I have tested this item :white_check_mark: successfully on 39b5ba8a8b5c57c7e542570068f8df53d351a625<br /><br />#pbf19 #test1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26910">issues.joomla.org/tracker/joomla-cms/26910</a>.</sub>

File: administrator/language/en-GB/en-GB.com_workflow.ini [hunk 1]
```diff
@@ -34,7 +34,7 @@ COM_WORKFLOW_FIELD_CATEGORY_LIST_LABEL="Category List"
 COM_WORKFLOW_FIELD_IS_DEFAULT_DESC="If set to default, this will be saved for the component."
 COM_WORKFLOW_FIELD_IS_DEFAULT_LABEL="Default"
 COM_WORKFLOW_FIELD_NAME_LABEL="Name"
-COM_WORKFLOW_FILTER_SEARCH_DESC="Filter the list of items."
+COM_WORKFLOW_FILTER_SEARCH_DESC="Search in title, description"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.com_workflow.ini]:
```suggestion
COM_WORKFLOW_FILTER_SEARCH_DESC="Search in title and description."
```

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**